src/theVoice/index.tsx

```diff
@@ -5,7 +5,7 @@ import TheVoice from './TheVoice'
 
 const StyledTheVoice = styled(TheVoice)`
   margin: 0;
-  padding: ${space.xl} 0 calc(${space.xl} + ${space.m}); /* 32px it's not en scale but on spec :() */
+  padding: ${space.xl} 0 calc(${space.xl} + ${space.m}); /* 32px it's not on scale but on spec :() */
```

Comment:
I would just use 32 px here or create a new constant.
You are doing arithmetic on things that are not semantically related: E.g. if someone is modifying the space.m constant one day, I'm not sure he would expect to modify the voice bottom padding at the same time.

I don't feel too strongly about and can keep that too since probably nobody will ever change these values globally. It just look like you have relationships between constants but actually we have none. The only relationship we have between these values is that they are a multiple of a base grid unit size (e.g. m => 1 grid unit (8px), xl => (2 gu -> 24px), etc).